include/sys/dsl_deleg.h

```diff
@@ -48,7 +48,6 @@ extern "C" {
 #define	ZFS_DELEG_PERM_RECEIVE		"receive"
 #define	ZFS_DELEG_PERM_ALLOW		"allow"
 #define	ZFS_DELEG_PERM_USERPROP		"userprop"
-#define	ZFS_DELEG_PERM_VSCAN		"vscan"
```

Comment:
it is example for future use on DilOS and can be done on others places in shared code
```
#ifdef __dilos__
#define	ZFS_DELEG_PERM_VSCAN		"vscan"
#endif
```